server/reflector/views/daily.py

```diff
@@ -0,0 +1,227 @@
+import json
+from typing import Any, Dict, Literal
+
+from fastapi import APIRouter, HTTPException, Request
+from pydantic import BaseModel
+
+from reflector.db.meetings import meetings_controller
+from reflector.logger import logger
+from reflector.settings import settings
+from reflector.utils.daily import DailyRoomName
+from reflector.video_platforms.factory import create_platform_client
+from reflector.worker.process import process_multitrack_recording
+
+router = APIRouter()
+
```

Comment:
```
from reflector.logger import logger as _logger
logger = _logger.bind(platform="daily")
```

This will prevent some logger passing daily and others no